### PR TITLE
Updated package readme

### PR DESCRIPTION
**Pull request by elotech47**
I updated the package readme to have a brief description of the package, how to install it and how to use it.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# ARRAY SCALER
+
+## Introduction
+A simple python tool for scaling and resvaling arrays:
```
Comment (DavidAkinpelu) [inline README.md]: I am not sure I know what `resvaling` is or is it a typo?

**Review by DavidAkinpelu — COMMENTED** [review]
I like your feature request. I provided a comments on something that seems like a typo.

File: README.md (hunk 1 above)
Comment (elotech47) [reply README.md]: Good catch!!!. This is actually a typo and I would fix it

Comment (elotech47) [inline README.md]: Fixed the typo issue here
```suggestion
A simple python tool for scaling and rescaling arrays:
```

**Pull request merged**